charts/datadog/values.yaml

```diff
@@ -431,6 +431,35 @@ datadog:
   #  - redisdb
   #  - kubernetes_state
 
+  # datadog.containerExclude -- Exclude containers from the Agent Autodiscovery
+  ## ref: https://docs.datadoghq.com/agent/guide/autodiscovery-management/?tab=containerizedagent#exclude-containers
+  containerExclude:  # "name:datadog/agent"
+
+  # datadog.containerInclude -- Include containers in the Agent Autodiscovery.
+  # If a container matches an include rule, it’s always included in the
+  # Autodiscovery
+  ## ref: https://docs.datadoghq.com/agent/guide/autodiscovery-management/?tab=containerizedagent#include-containers
+  containerInclude:
+
+  # datadog.containerExcludeLogs -- Exclude logs from the Agent Autodiscovery
+  containerExcludeLogs:
+
+  # datadog.containerIncludeLogs -- Include logs in the Agent Autodiscovery
+  containerIncludeLogs:
+
+  # datadog.containerExcludeMetrics -- Exclude metrics from the Agent
+  # Autodiscovery
+  containerExcludeMetrics:
+
+  # datadog.containerIncludeMetrics -- Include metrics in the Agent
+  # Autodiscovery
+  containerIncludeMetrics:
```

Comment:
It can be useful if we remind users that these fields are supposed to be space separated strings